pkgs/by-name/tw/tweego/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "tweego";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "tmedwards";
+    repo = "tweego";
+    rev = "v${version}";
+    hash = "sha256-LE85mSByTz7uFjs0XtrpfD7OARoMPE56FpjFw+FlGYw=";
+  };
+
+  proxyVendor = true;
+  vendorHash = "sha256-1O27CiCXgrD0RC+3jrVxAiq/RnI2s1FW2/uoBAd1fF8=";
+
+  preBuild = ''
+    go mod tidy
+  '';
+
+  meta = {
+    description = "Tweego is a free (gratis and libre) command line compiler for Twine/Twee story formats, written in Go.";
```

Comment:
```suggestion
    description = "Free (gratis and libre) command line compiler for Twine/Twee story formats, written in Go";
```

Reply:
Thanks! I've re-ordered the commits (updating code as necessary) and implemented your suggestion. Makes a lot of sense to keep to the point.